Test main

AbbyW1212 opened the pull request:
- added in coral elevator/endeffector mechanisms and commands
- updated readme
- fixed swerve module CAN IDs to match readme
- vision and localization fixes

AbbyW1212 committed:
Got lift/claw/elevator/endeffector working + commands

Mindi088 committed:
CoralElevatorToHeight Command working

Angelina (and Abby)

AbbyW1212 committed:
Updated README, created cancel load and trough score command

arjunio101 committed:
localize works

arjunio101 committed:
Merge branch 'vision-fixes' into test-main

PurpleToaster83 committed:
added the trigger for alignToTag (untested)

added a trigger, import, and instance of aprilTagAlign to RobotContainer. Untested (if not work reverting to prior will simply make alignToTag not work with no other adverse affects)

AngelinaCosta reviewed (approved):
This has the William and Angelina seal of approval.

Mindi088 reviewed (approved):
👍

Mindi088 committed:
updated README to add driver controller

Angelina, Abby, and Will

nsargent1073 committed:
wpilib 02-14 update ruined everything

new wpilib update to 2025.3.1 on 02/14/2025. committing updates to three metadata files and FieldMap.java. the reefscape field JSON now has two versions, AndyMark and welded, and 'k2025Reefscape' is not a real variable anymore.

AbbyW1212 committed:
Merge branch 'main' into test-main
